### PR TITLE
Adding livepeer livestream page

### PR DESCRIPTION
**Pull request by rahulrumalla**
For issue #19

### CODE REVIEW

**Comment by apollonian11** [comment]
Thanks a lot for the PR @rahulrumalla! 😻

File: livestream/index.html [hunk 1]
```diff
@@ -0,0 +1,313 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <title>Tickets - ETHIndia | August 10 - 12, 2018</title>
```
Comment (apollonian11) [inline livestream/index.html]: Would recommend a few changes before we merge this to master.

[ ] Changing the title here to `Livestream - ETHIndia | August 10 - 12, 2018`

File: livestream/index.html [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+    </footer>
+  </section>
+
+  <!-- Bootstrap core JavaScript -->
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.0.0/js/bootstrap.bundle.min.js"></script>
+  <!-- Custom scripts -->
+  <script src="../js/main.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/bodymovin/4.13.0/bodymovin.min.js"></script>
+  <script src="https://checkout.razorpay.com/v1/checkout.js"></script>
```
Comment (apollonian11) [inline livestream/index.html]: [ ] Getting rid of the unused code in the page specifically #L147-#L284

**Comment by apollonian11** [comment]
@rahulrumalla Thanks! 🖖

**Comment by chrishobcroft** [comment]
Great guys - what's the next step for releasing this?

**Comment by chrishobcroft** [comment]
I have a question about the parameters used in the iframe:

```
width="1100" height="720"
```

The original request was for

```
width="1280" height="720"
```

If this doesn't work with the size of your screen, then let's make it:

```
width="854" height="480"
```

**Comment by decentralizedcultureroom** [comment]
Is there somewhere I can see it in action?

**Comment by chrishobcroft** [comment]
Is there somewhere I can see it in action?

**Comment by apollonian11** [comment]
@chrishobcroft I've deployed it here: https://ethindia.surge.sh

**Comment by rahulrumalla** [comment]
@apollonian11 is the CI hooked up for dev/staging?

If not, can you re-deploy with my recent commit, please?

**Comment by chrishobcroft** [comment]
Looks good to me in Chrome 68.0.3440.84 and Safari Version 11.1.2 (13605.3.8) on MacOS High Sierra.

It doesn't seem to work on Brave - but I think that's a Livepeer issue.

**Comment by rahulrumalla** [comment]
The glitch problem persists across Chrome, Opera & Brave on my end. 
For the page to work on brave browser, you need to allow third-party cookies (option: Allow all cookies) on your settings. 

What is the team's suggestion / preference on moving ahead on this?

**Comment by chrishobcroft** [comment]
I would like to release this as it is to ethindia.co

@apollonian11 can we please push this to the live site?

**Comment by apollonian11** [comment]
@chrishobcroft Will do. The player is no longer glitching on my laptop. I'll still see if change in the dimensions is helpful.

**Comment by apollonian11** [comment]
@rahulrumalla @chrishobcroft Merging this to master. 🎉

**Pull request merged**